src/services/Api.tsx

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+export const weatherData = axios.create({
```

Comment:
Is this being used? looking at the `Weather.tsx` component I can't see `weatherData` being referenced?